packages/react-server/src/ReactFlightServer.js

```diff
@@ -1306,7 +1350,7 @@ function renderModelDestructive(
       }
       if (value instanceof Float64Array) {
         // double
-        return serializeTypedArray(request, 'D', value);
+        return serializeTypedArray(request, 'd', value);
```

Comment:
I switched this to use the lower case since the capital version fits better with the other common cases and this is very rare.

(The d is for double.)